aws/src/main/java/org/apache/iceberg/aws/AwsProperties.java

```diff
@@ -213,13 +213,20 @@
    * Enables eTag checks for S3 PUT and MULTIPART upload requests.
    */
   public static final String S3_CHECKSUM_ENABLED = "s3.checksum-enabled";
+
+  /**
+   * Sets the size for batch deletion.
+   */
+  public static final int S3_DELETION_BATCH_SIZE_DEFAULT = 10000;
```

Comment:
should be 1000 instead of 10000

Reply:
nit, might worth putting a link to s3 documentation to reference where is 1000 come from https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObjects.html

Reply:
Very true, I actually should not assume that it's a good default until I do some benchmarking.